compiler/extccomp.nim

```diff
@@ -994,6 +1047,12 @@ proc callCCompiler*(conf: ConfigRef) =
         preventLinkCmdMaxCmdLen(conf, linkCmd)
         for cmd in extraCmds:
           execExternalProgram(conf, cmd, hintExecuting)
+        # create Mac app bundle for GUI apps on macOS
+        when defined(macosx):
+          if optGenGuiApp in conf.globalOptions and
+             optGenDynLib notin conf.globalOptions and
+             optGenStaticLib notin conf.globalOptions:
```

Comment:
```suggestion
          if conf.globalOptions * {optGenGuiApp, optGenDynLib, optGenStaticLib} == {openGenGuiApp}:
```

Reply:
Ah, that `openGenGuiApp` typo got me searching really hard :)